libr/bin/p/bin_vsf.c

```diff
@@ -96,6 +104,11 @@ static bool load(RBinFile *bf, RBuffer *b, ut64 loadaddr) {
 			} else if (!CMP_MODULE (VICE_MAINCPU) && module.major == 1) {
 				res->maincpu = R_NEW (struct vsf_maincpu);
 				r_buf_read_at (bf->buf, offset + read, (ut8 *)res->maincpu, sizeof (*res->maincpu));
+			} else {
+				char safe_name[sizeof (module.module_name) + 1];
+				memcpy (safe_name, module.module_name, sizeof (module.module_name));
```

Comment:
```suggestion
				memcpy (safe_name, module.module_name, sizeof (safe_name));
```

its better to use the sizeof of the destination even if both are the same